### PR TITLE
Prepare release of 0.4.0

### PR DESCRIPTION
**Pull request by webwurst**
This PR:

- adds/changes/removes etc

### Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/efk-stack-app/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,5 @@ sources:
 - https://raw.githubusercontent.com/giantswarm/efk-stack-app/v[[ .Version ]]/README.md
 annotations:
   application.giantswarm.io/values-schema: https://raw.githubusercontent.com/giantswarm/efk-stack-app/v[[ .Version ]]/helm/efk-stack-app/values.schema.json
-version: [[ .Version ]]
+# version: [[ .Version ]]
```
Comment (ubergesundheit) [inline helm/efk-stack-app/Chart.yaml:29]:
```suggestion
```

File: helm/efk-stack-app/charts/opendistro-es/Chart.yaml [hunk 2]
```diff
@@ -26,4 +26,5 @@ name: opendistro-es
 sources:
 - https://pages.git.viasat.com/ATG/charts
 # Chart version
-version: 1.9.0
+version: [[ .Version ]]
+# icon: https://avatars1.githubusercontent.com/u/45981434
```
Comment (ubergesundheit) [inline helm/efk-stack-app/charts/opendistro-es/Chart.yaml:30]:
```suggestion
```

**Comment by webwurst** [comment]
When I run the failing lint check locally I get a different error, probably because of newer version:
```
$ docker run -v $PWD:$PWD -w $PWD quay.io/helmpack/chart-testing:v3.3.1 ct lint --validate-maintainers=false --charts="helm/efk-stack-app"

==> Linting helm/efk-stack-app
[ERROR] /home/webwurst/Development/github.com/giantswarm/efk-stack-app/helm/efk-stack-app: chart metadata is missing these dependencies: opendistro-es,elasticsearch-curator,elasticsearch-exporter,fluentd-elasticsearch,opendistro-certs
```
Just sharing, will look more into this, also try older version.

**Comment by webwurst** [comment]
The circle-ci step "start kube-app-testing tool" is failint with this error:
```
Error: error initializing: Looks like "https://kubernetes-charts.storage.googleapis.com" is not a valid chart repository or cannot be reached: Failed to fetch https://kubernetes-charts.storage.googleapis.com/index.yaml : 403 Forbidden
```
Maybe this is because something was deprecated around repository structure and helm2? I get the same error locally when running `chart-testing:v2.4.1`. Version `v3.3.1` does not show this error, but expects other changes to make the chart compatible with helm3.

**Comment by ubergesundheit** [comment]
could you try `fix-helm-url` branch of `kube-app-testing`?

**Comment by webwurst** [comment]
Superseded by https://github.com/giantswarm/efk-stack-app/pull/42

**Pull request closed without merging**